docs/design/autoware-architecture/perception/index.md

```diff
@@ -1,5 +1,97 @@
+
+In our overall design, we emphasize the concept of [microautonomy architecture](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-concepts). This term refers to a design approach that focuses on the proper modularization of functions, clear definition of interfaces between these modules, and as a result, high expandability of the system. Given this context, the goal of the Perception component is set not to solve every conceivable complex use case (although we do aim to support basic ones), but rather to provide a platform that can be customized to the user's needs and can facilitate the development of additional features.
+
+To clarify the design concepts, the following points are listed as goals and non-goals.
+
+**Goals:**
+
+- The basic functions are provided so that a simple ODD can be defined.
+- To achieve a design that can provide Perception functionality to every autonomous vehicle.
+- The capability is extensible with the third-party components.
+- The Perception component is designed to provide a platform that enables autoware users to develop the complete functionality and capability.
+- The Perception component is designed to provide a platform that enables autoware users to develop the autonomous driving system which always outperforms human drivers.
+- The Perception component is designed to provide a platform that enables autoware users to develop the autonomous driving system achieving "zero overlooks" or "error-free recognition".
+
+**Non-goals:**
+
+- The Perception Component should not operate perfectly only in specific environments and be completely useless in other environments.
+- The Perception component is not self-contained but can be extended with third parties.
+- The Perception component is not aimed at the complete functionality and capability.
+- The Perception component is not designed to always outperform human drivers.
+- The Perception component is not capable of achieving "zero overlooks" or "error-free recognition".
+
+## High-level architecture
+
+This diagram describes the high-level architecture of the Perception Component.
+
+![overall-perception-architecture](image/high-level-perception-diagram.drawio.svg)
+
+The Perception component consists of the following sub-components:
+
+- **Object Recognition**: Recognizes dynamic objects surrounding the ego vehicle in the current frame and predicts their future trajectories.
+- **Obstacle Segmentation**: Identifies point clouds originating from obstacles(not only dynamic objects but also static obstacles that should be avoided, such as stationary obstacles) that the ego vehicle should avoid. For example, construction cones are recognized using this module.
```

Comment:
IMO the modules responsible for recognizing construction cones are open to debate, and some may argue that it should be handled in object recognition module for instance.
I believe it is better to remove this sentence from this "High-level architecture" page.

```suggestion
- **Obstacle Segmentation**: Identifies point clouds originating from obstacles(not only dynamic objects but also static obstacles that should be avoided, such as stationary obstacles) that the ego vehicle should avoid.
```